go/vt/sqlparser/parser.go

```diff
@@ -35,6 +35,9 @@ var parserPool = sync.Pool{}
 // zeroParser is a zero-initialized parser to help reinitialize the parser for pooling.
 var zeroParser = *(yyNewParser().(*yyParserImpl))
 
+// MySQLVersion is the version of MySQL that the parser would emulate
+var MySQLVersion string = "5.7.9"
+
```

Comment:
VtGate mysql version is used, I think this is redundant and can be removed.

Reply:
We cannot use Vtgate mysql version directly since it creates an import cycle. So we set this variable in the parser from the vtgate.